scripts/zones/Ceizak_Battlegrounds/mobs/Transcendent_Scorpion.lua

```diff
@@ -0,0 +1,61 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Transcendent Scorpion
+-- !pos 160 0 -400 261
+-- !additem 6013
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+local removables = {
+xi.effect.FLASH, xi.effect.BLINDNESS, xi.effect.ELEGY, xi.effect.REQUIEM,
+xi.effect.PARALYSIS, xi.effect.POISON, xi.effect.DISEASE, xi.effect.PLAGUE,
+xi.effect.WEIGHT, xi.effect.BIND, xi.effect.BIO, xi.effect.DIA, xi.effect.BURN,
+xi.effect.FROST, xi.effect.CHOKE, xi.effect.RASP, xi.effect.SHOCK, xi.effect.DROWN,
+xi.effect.STR_DOWN, xi.effect.DEX_DOWN, xi.effect.VIT_DOWN, xi.effect.AGI_DOWN,
+xi.effect.INT_DOWN, xi.effect.MND_DOWN, xi.effect.CHR_DOWN, xi.effect.ADDLE, xi.effect.SLOW,
+xi.effect.HELIX, xi.effect.ACCURACY_DOWN, xi.effect.ATTACK_DOWN, xi.effect.EVASION_DOWN,
+xi.effect.DEFENSE_DOWN, xi.effect.MAGIC_ACC_DOWN, xi.effect.MAGIC_ATK_DOWN,
+xi.effect.MAGIC_EVASION_DOWN, xi.effect.MAGIC_DEF_DOWN, xi.effect.MAX_TP_DOWN,
+xi.effect.MAX_MP_DOWN, xi.effect.MAX_HP_DOWN
+}
+
+local function removeSpecificDebuffs(mob)
+local numEffectsRemoved = 0
+for _, effect in ipairs(removables) do
+    if mob:hasStatusEffect(effect) then
+        mob:delStatusEffect(effect)
+        numEffectsRemoved = numEffectsRemoved + 1
+    end
+end
+
+return numEffectsRemoved
+end
+
+entity.onMobFight = function(mob, target)
+if mob:getHPP() <= 50 then
```

Comment:
This needs indenting

Reply:
This is also going to be setting the skill list every tick - which is harmless, but worth knowing for the future